dist/images/daemonset.sh

```diff
@@ -232,7 +242,15 @@ while [ "$1" != "" ]; do
 done
 
 # Create the daemonsets with the desired image
-# They are expanded into daemonsets in ../yaml
+# They are expanded into daemonsets in the specified
+# output directory.
+if [ -d ${OVN_OUTPUT_DIR} ]; then
+  output_dir=${OVN_OUTPUT_DIR}
+else 
+  output_dir=${OVN_OUTPUT_DIR}
```

Comment:
no reason to set output_dir twice? shouldn't you also default the output_dir to ../yaml like it used to be?

Reply:
Yep fixed the logic here to default  to `../yaml` if `OVN_OUTPUT_DIR` is not set, and make the DIR if `OVN_OUTPUT_DIR` doesn't exist